libsolidity/ast/Types.cpp

```diff
@@ -1578,8 +1578,6 @@ bool ArrayType::canBeUsedExternally(bool _inLibrary) const
 		return true;
 	else if (!m_baseType->canBeUsedExternally(_inLibrary))
 		return false;
-	else if (m_baseType->category() == Category::Array && m_baseType->isDynamicallySized())
-		return false;
```

Comment:
So basically we enable multi level arrays now for non-experimental mode. Will it be still disabled due to no support for returndatacopy? Do we have a test for this?

Reply:
It is still disabled by the type checker.

Reply:
The function `typeSupportedByOldABIEncoder` is much stricter than what was there before.